numba/cuda/tests/cudasim/test_cudasim_issues.py

```diff
@@ -1,14 +1,49 @@
 import threading
 
 import numpy as np
+from numpy import float64
 
-from numba import cuda
+from numba import cuda, numba
```

Comment:
Am I reading this right, implies `from numba import numba`?

Reply:
Will swap to something better.